src/main/java/core/basesyntax/dp/Storage.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dp;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> storage = new HashMap<>();
```

Comment:
The `storage` field is public and static, which can lead to issues with encapsulation and potential thread safety problems if accessed concurrently. Consider using private access with public getter methods or synchronized methods to manage access if thread safety is a concern.